domain/schema/model.go

```diff
@@ -44,3 +45,25 @@ CREATE TABLE model_config (
 );
 `)
 }
+
+func spacesSchema() schema.Patch {
+	return schema.MakePatch(`
+CREATE TABLE provider_spaces (
+    uuid            TEXT PRIMARY KEY,
```

Comment:
These fields should be `provider_id` and `space_uuid`.

It doesn't matter which you make the primary key, because they both have to be unique within the model, but there should be a unique index on the _other_.